selfdrive/test/test_routes.py

```diff
@@ -372,6 +372,11 @@
     'enableCamera': True,
     'enableDsu': False,
   },
+  "3fd5305f8b6ca765|2021-04-28--12-28-19": {
```

Comment:
this route isn't fully uploaded. you'll need to connect your device to wifi for it to upload

Reply:
I reran the test route and updated the run name.